bin/diff_yaml.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+# Compare 2 multi document kubernetes yaml files
+# It ensures that order does not matter
+
+import yaml  # pyyaml
+import datadiff
+import argparse
+
+# returns fully qualified resource name of the k8s resource
+
+
+def byResourceName(res):
+    if res is None:
+        return ""
+
+    return "{}::{}::{}".format(res['apiVersion'],
+                               res['kind'],
+                               res['metadata']['name'])
+
+
+def keydiff(k0, k1):
+    k0s = set(k0)
+    k1s = set(k1)
+    added = k1s - k0s
+    removed = k0s - k1s
+    common = k0s.intersection(k1s)
+
+    return added, removed, common
+
+
+def drop_keys(res, k1, k2):
+    if k2 in res[k1]:
+        del res[k1][k2]
+
+
+def normalize_configmap(res):
+    if res['kind'] != "ConfigMap":
```

Comment:
typically references should have exception handling  - but since this is a simple tool not meant for users to really run, likely doesn't matter.